### PR TITLE
[Resolves #27 and #36] associate users with reviews / show lifehacks 

### LINKED ISSUE #5: show Lifehack
As a User
I want to see all details about a single lifehack
So I can learn a new hack

```
Acceptance Criteria:
```
- [ ] I should be able to click on a Lifehack list item on the index page to be brought to the show page for that individual Lifehack
- [ ] I should be able to see all details about the lifehack
- [ ] I should be able to see all reviews of the lifehack
- [ ] I should be able to sort reviews by votes and date
- [ ] I should be able to click a link to navigate back to the index page

### LINKED ISSUE #27: associate users with reviews

### PR DESCRIPTION
**Pull request by Tko176**

### CODE REVIEW

**Comment by Tko176** [comment]
#27 #36

File: db/seeds.rb [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
+#     last_name: 'Knot',
+#     email: 'jon@gmail.com',
+#     password: 'whattheheck?' }]
+#                )
+# Lifehack.create([{
+#     title: 'How to tie a tie',
+#     description: 'tie a knot',
+#     creator_id: "#{User.first.id}"  }, {
+#     title: 'How to tie a tie1',
+#     creator_id: "#{User.first.id}" }])
```
Comment (EliseFitz15) [inline db/seeds.rb:20]: If you are no longer using these you should delete them.

File: spec/factories/review_factory.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'factory_girl_rails'
+
+FactoryGirl.define do
+  factory :review do
+    rating 5
+    sequence(:comment) { |n| "Best lifehack ever#{n}" }
+    association :lifehack
+    association :creator, factory: :user
+  end
```
Comment (EliseFitz15) [inline spec/factories/review_factory.rb:9]: What's great about factory girl is as long as you have a factory for all of your objects you can actually just change lines 7 and 8 to be: 

```
lifehack
creator
```

File: app/models/review.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class Review < ActiveRecord::Base
   belongs_to :lifehack
+  belongs_to :creator, class_name: 'User'
   RATING_LIST = [1, 2, 3, 4, 5]
```
Comment (EliseFitz15) [inline app/models/review.rb:4]: move global values to line two of the model

**Pull request merged**